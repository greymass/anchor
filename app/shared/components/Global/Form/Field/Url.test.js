import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import GlobalFormFieldUrl from './Url';

const mockProps = {
  disabled: false,
  error: null,
  fluid: true,
  icon: 'icon',
  inline: true,
  label: 'multi token field',
  loading: false,
  name: 'multi token field',
  placeholder: 'string placeholder',
};

describe('GlobalFormFieldUrl', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<GlobalFormFieldUrl {...mockProps} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
