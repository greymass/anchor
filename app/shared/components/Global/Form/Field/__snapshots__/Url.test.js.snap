// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalFormFieldUrl renders correctly 1`] = `
<FormField
  control={[Function]}
  disabled={false}
  error={null}
  fluid={true}
  icon="icon"
  inline={true}
  label="multi token field"
  loading={false}
  name="multi token field"
  onChange={[Function]}
/>
`;
